src/main/java/com/leverx/mediator/dto/converter/Mapper.java

```diff
@@ -0,0 +1,16 @@
+package com.leverx.mediator.dto.converter;
+
+import static lombok.AccessLevel.PRIVATE;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import lombok.NoArgsConstructor;
+
+/**
+ * @author Andrei Yahorau
+ */
+@NoArgsConstructor(access = PRIVATE)
+public final class Mapper {
+
+  public static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
Could you please create more 'humanity named' methods here to convert from jsonString and vice versa?